src/qmapshack/gis/fit/decoder/IFitDecoderState.h

```diff
@@ -84,8 +84,9 @@ class IFitDecoderState
     quint32 getTimestamp() const { return data.timestamp; }
     quint16 getCrc() const { return data.crc; }
     void addDevFieldProfile(const CFitFieldProfile &fieldProfile);
-    CFitFieldProfile* devFieldProfile(quint32 fieldNr);
-    void clearDevFieldProfiles();
+    CFitFieldProfile* devFieldProfile(QPair<quint8,quint8> devProfileId);
```

Comment:
no `const QPair<quint8,quint8>&` ?

Reply:
Yes, will do.

Reply:
```
CFitFieldProfile* devFieldProfile(const QPair<quint8,quint8>& devProfileId);
```

you do not want to create another copy on the stack ;)

Reply:
Now I know why the code reviews works pretty well ;-)